src/System.ComponentModel.TypeConverter/tests/Drawing/ImageFormatConverterTests.cs

```diff
@@ -0,0 +1,225 @@
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void GetStandardValuesSupported()
+        {
+            Assert.True(_imgFmtConv.GetStandardValuesSupported(), "GetStandardValuesSupported()");
+            Assert.True(_imgFmtConv.GetStandardValuesSupported(null), "GetStandardValuesSupported(null)");
+        }
+
+        private void CheckStandardValues(ICollection values)
+        {
+            bool memorybmp = false;
+            bool bmp = false;
+            bool emf = false;
+            bool wmf = false;
+            bool gif = false;
+            bool jpeg = false;
+            bool png = false;
+            bool tiff = false;
+            bool exif = false;
+            bool icon = false;
+
+            foreach (ImageFormat iformat in values)
+            {
+                switch (iformat.Guid.ToString())
+                {
+                    case "b96b3caa-0728-11d3-9d7b-0000f81ef32e":
+                        memorybmp = true;
+                        break;
+                    case "b96b3cab-0728-11d3-9d7b-0000f81ef32e":
+                        bmp = true;
+                        break;
+                    case "b96b3cac-0728-11d3-9d7b-0000f81ef32e":
+                        emf = true;
+                        break;
+                    case "b96b3cad-0728-11d3-9d7b-0000f81ef32e":
+                        wmf = true;
+                        break;
+                    case "b96b3cb0-0728-11d3-9d7b-0000f81ef32e":
+                        gif = true;
+                        break;
+                    case "b96b3cae-0728-11d3-9d7b-0000f81ef32e":
+                        jpeg = true;
+                        break;
+                    case "b96b3caf-0728-11d3-9d7b-0000f81ef32e":
+                        png = true;
+                        break;
+                    case "b96b3cb1-0728-11d3-9d7b-0000f81ef32e":
+                        tiff = true;
+                        break;
+                    case "b96b3cb2-0728-11d3-9d7b-0000f81ef32e":
+                        exif = true;
+                        break;
+                    case "b96b3cb5-0728-11d3-9d7b-0000f81ef32e":
+                        icon = true;
+                        break;
+                    default:
+                        throw new InvalidOperationException($"Unknown GUID {iformat.Guid}.");
+                }
+            }
+            Assert.True(memorybmp, "MemoryBMP");
+            Assert.True(bmp, "Bmp");
+            Assert.True(emf, "Emf");
+            Assert.True(wmf, "Wmf");
+            Assert.True(gif, "Gif");
+            Assert.True(jpeg, "Jpeg");
+            Assert.True(png, "Png");
+            Assert.True(tiff, "Tiff");
+            Assert.True(exif, "Exif");
+            Assert.True(icon, "Icon");
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void GetStandardValues()
+        {
+            CheckStandardValues(_imgFmtConv.GetStandardValues());
```

Comment:
Instead of calling a helper method, this test can be a ConditionalTheory and pass the values as a [MemberData](https://github.com/dotnet/corefx/blob/efdbbe6ba587a9efb1389f6d090b564f441c7b86/src/System.Memory/tests/ParsersAndFormatters/Parser/ParserTests.cs#L12) or [InlineData](https://github.com/dotnet/corefx/blob/efdbbe6ba587a9efb1389f6d090b564f441c7b86/src/System.Memory/tests/ParsersAndFormatters/Parser/ParserTests.cs#L124)

Reply:
I am not sure, how to use `Theory` in this case.